rcon/commands.py

```diff
@@ -719,6 +719,18 @@ def get_gamestate(self) -> list[str]:
 
         return self._str_request("get gamestate", can_fail=False).split("\n")
 
+    def get_objective_row(self, row: int):
+        if not (0 <= row <= 4):
+            raise ValueError('Row must be between 0 and 4')
+        return self._get_list(f'get objectiverow_{row}')
+
+    def set_game_layout(self, objectives: Sequence[str]):
+        if len(objectives) != 5:
+            raise ValueError("5 objectives must be provided")
+        return (
```

Comment:
it might be a lot of commands here, but it might be worth doing a get_objective_row call for each row and check if the provided capture points _actually_ exists for the current map? 🤔

Especially as we cache the get objectiverow call for up to 10 seconds, which might then even be a new map already.

=> Nevermind, it's in rcon.py already 👍